windows/docker-compose.yml

```diff
@@ -45,6 +46,9 @@ services:
       - SITE_ROOT=api
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
```

Comment:
we might need to use the override approach https://docs.docker.com/compose/extends/#example-use-case
normal without the config ENV variable and the override with it.